libraries/FreeRTOS/FreeRTOS.ino

```diff
@@ -0,0 +1,114 @@
+#if CONFIG_FREERTOS_UNICORE
+#define ARDUINO_RUNNING_CORE 0
+#else
+#define ARDUINO_RUNNING_CORE 1
+#endif
+
+// define two tasks for Blink & AnalogRead
+void TaskAnalogReadA0( void *pvParameters );
+void TaskAnalogReadA3( void *pvParameters );
+
+// the setup function runs once when you press reset or power the board
+void setup() {
+  
+  // initialize serial communication at 9600 bits per second:
+  Serial.begin(115200);
+  
+  while (!Serial) {
+    ; // wait for serial port to connect. Needed for native USB, on LEONARDO, MICRO, YUN, and other 32u4 based boards.
+  }
+
+  // Now set up two tasks to run independently.
+  xTaskCreatePinnedToCore(
+    TaskAnalogReadA0
+    ,  "AnalogReadA0"   // A name just for humans
+    ,  128  // This stack size can be checked & adjusted by reading the Stack Highwater
```

Comment:
In esp-idf docs stack highwater is wrong description. Vanilla FreRTOS is using dword for stack size, but esp32 stack size is in bytes, so 128 bytes i way to small stack. I would start with 1kB just in case.